### DISCUSSION TITLE
Does 'signal send' blocks current execution?

### DISCUSSION THREAD

**Discussion by kashcode** [pallets-eco/blinker#82] [Q&A] [answered]
I wondering if 'signal send' blocks current execution, for example I send signal for some action 'signal handler' handle and do some actions, does this action is fire and forget?

**Comment by pawamoy** [comment, marked as answer]
> Does 'signal send' blocks current execution?

Yes. `your_signal.send(...)` returns a list of tuples: the receivers and their return value (if any).
